mersenneprime.java

```diff
@@ -0,0 +1,46 @@
+//To find mersenne's prime upto 
+

```

Comment:
Please briefly describe what is Mersenne's Prime number.

Reply:
Also, it would be great if you could add a sample input and output as comments to make it more clear.